src/main/java/Controllers/ReviewManager.java

```diff
@@ -0,0 +1,61 @@
+package Controllers;
+
+import Entities.Review;
+import UseCaseClasses.InformationSaver;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+public class ReviewManager {
+
+    private HashMap<String, Review> reviews;
+
+    public ReviewManager() {
+        reviews = new HashMap<>();
+    }
+
+    /**
+     *
+     * @param Location The location that the review is for.
+     * @param reviewBody The body of the review.
+     * @param rating The score out of 5 given by the reviewer.
+     * @return true if the review was successfully created and added
+     */
+
```

Comment:
pretty sure javadocs aren't supposed to have a space between the method header and the last */ thing?